src/site/content/en/blog/accent-color/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: CSS `accent-color`
+subhead: Bring your brand color to built-in HTML form inputs with one line of code.
+authors:
+  - adamargyle
+description: Bring your brand color to built-in HTML form inputs with one line of code.
+date: 2021-08-09
+thumbnail: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/WOcuCLCwMr0M2lF17bmm.png
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/huEpiCoJQ6dAo8rHGsZT.png
+tags:
+  - blog
+  - css
+---
+
+Today's HTML form elements are [difficult to
+customize](https://codepen.io/GeoffreyCrofte/pen/BiHzp). It feels as if it's
+either: no custom styles, very few custom styles, or reset input styles and
+build it up from scratch. Building it up from scratch ends up being much more
+work than anticipated; forgotten styles for element states
+([indeterminate](https://developer.mozilla.org/en-US/docs/Web/CSS/:indeterminate),
+I'm looking at you), losing built in accessibility features, etc. It may be more
+work than you're looking to take on.
```

Comment:
```suggestion
Today's HTML form elements are [difficult to
customize](https://codepen.io/GeoffreyCrofte/pen/BiHzp). It feels as if it's
a choice between few or no custom styles, or resetting input styles and
build it up from scratch. Building it up from scratch ends up being much more
work than anticipated. It can also lead to forgotten styles for element states
([indeterminate](https://developer.mozilla.org/en-US/docs/Web/CSS/:indeterminate),
I'm looking at you), and the loss of built-in accessibility features. 
To fully recreate what the browser provides may be more
work than you're looking to take on.
```